Gruntfile.js

```diff
@@ -0,0 +1,180 @@
+module.exports = function(grunt) {
```

Comment:
it can but you would need to take care also of browserify, mocha, instambul, connect etc.

im all that i'm suggesting to you on GlobaLeaks, scrypt-async, openpgp.js, flow.js and s it is really audited and functional, but feel free to implement it in a different way if you prefer.